test/spec.js

```diff
@@ -388,6 +388,66 @@ describe('checkDependencies', function () {
             });
         });
 
+        it('should check custom package name dependencies only if `checkCustomPackageNames` is true ' +
+           'and we are testing bower, not npm', function (done) {
+
+            if (packageManager !== 'npm') {
+              checkDeps({
+                  checkCustomPackageNames: true,
+                  packageDir: './test/bower-fixtures/custom-package',
+                  scopeList: ['dependencies', 'devDependencies'],
+              }, function (output) {
+                  assert.deepEqual(output.error, [
+                      'a: installed: 0.5.8, expected: 0.5.9',
+                      'b: not installed!',
+                      installMessage,
+                  ]);
+              });
+
+              checkDeps({
+                  packageDir: './test/bower-fixtures/custom-package',
+                  scopeList: ['dependencies', 'devDependencies'],
+              }, function (output) {
+                  assert.deepEqual(output.error, [
+                      'b: not installed!',
+                      installMessage,
+                  ]);
+                  done();
+              });
+
+            } else {
+              done();
+            }
+
+        });
+
+        it('should check the version for custom package names valid semver tags only', function (done) {
+            checkDeps({
```

Comment:
Ah - I think you are right; the 'non-semver-tag' tests for checkGitUrls=true already cover the code that checks for package existence regardless of version number, so these two of my tests are unnecessary.

I had originally intended checkCustomPackageNames to be feature that solved a more general case of the issue solved by checkGitUrls, but NPM doesn't (yet) support custom package names like bower does, so it can't do that right now.

I removed these two tests in my next push.